src/components/Button/index.tsx

```diff
@@ -65,6 +66,7 @@ export const Button = ({
     isLoading = false,
     rounded = "sm",
     size = "md",
+    hasHoverStyle=true,
```

Comment:
IMO i think its better to have it set to false first and rename it to hideHoverStyle. This is because in terms of simplicity of when you want to pass this prop to a component it will be a bit more readable and you don't need to pass `false` every time we want to hide the hover styles. For example here are the two differences:

<Button hasHoverStyles={false} /> vs <Button hideHoverStyles />

The second example is more straightforward and will handle setting it to true by simply passing the prop

```suggestion
    hideHoverStyles=false,
```

Reply:
Yes This is more readable.I will make changes.Thanks